react-components/src/components/Architecture/CommandButton.tsx

```diff
@@ -67,10 +67,15 @@ export const CommandButton = ({
   }
   const placement = isHorizontal ? 'top' : 'right';
   const { key, fallback } = newCommand.tooltip;
+  // This was the only way it went through compiler: (more bytton types will be added in the future)
+  const type = newCommand.buttonType;
+  if (type !== 'ghost' && type !== 'ghost-destructive') {
+    return <></>;
+  }
```

Comment:
Just curious - what kind of error do you get when you don't have this?

Reply:
I try to cast, but ButtonType/GhostButtonType is not exposed. It has to be a ButtonType or a GhostButtonType. The syntax i cogs is really hard to understand.